spec/spec_helper.rb

```diff
@@ -13,3 +13,16 @@
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 require 'google_maps_geocoder/google_maps_geocoder'
+# silence output
+RSpec.configure do |config|
+  config.before(:example, silence_logger = true) do
+    if silence_logger
+      quiet_logger = Logger.new(IO::NULL)
+      allow(Logger).to receive(:new).and_return(quiet_logger)
+    end
```

Comment:
```suggestion
  config.before(:example) do
    quiet_logger = Logger.new(IO::NULL)
    allow(Logger).to receive(:new).and_return(quiet_logger)
```

I guess we don't need a metadata setting.

Reply:
Good point!